README.md

```diff
@@ -1,3 +1,101 @@
-# MoveSingleAPISwift
+# MoveSingleAPI SDK in Swift
 
-A description of this package.
+## Installation 💻
+You can use Swift Package Manager to integrate the library by adding the following dependency in the `Package.swift` file or by adding it directly within Xcode.
+
+`.package(url: "https://github.com/move-ai/MoveSingleAPISwift", from: "1.0.0")`
+
+## Usage 🤩
+Import the framework in your project:
+```swift
+import MoveSingleAPISwift
+```
+
+[Create an MoveSingleAPI key](https://move.ai) and add it to your configuration:
+```swift
+let move = Move(apiKey: "TOKEN")`
+```
+
+### Create simple Take
```

Comment:
Can we add a line explaining what a take is, using the copy from the Python SDK : 
"Takes are a way to define a recording session. A single take can be associated with a video file and optionally additional files."

Reply:
I change it to "additional enhancement data" to make it more clear.